apps/desktop/desktop_native/windows-plugin-authenticator/Cargo.toml

```
@@ -0,0 +1,7 @@
+[package]
+name = "windows-plugin-authenticator"
+version = "0.1.0"
+edition = "2021"
```

Updated this to be more inline with the other crates in the worskpace

```suggestion
name = "windows-plugin-authenticator"
version = "0.0.0"
edition = "2021"
license = "GPL-3.0"
publish = false
```

---

Addressed in https://github.com/bitwarden/clients/pull/13112/commits/65481bc15baa831f4a085dfc32bff142249317f8